app/assets/stylesheets/hyrax/_work-show.scss

```diff
@@ -40,3 +40,29 @@ ul.tabular {
   padding: $panel-body-padding;
   // text-align: center;
 }
+
+.table-borderless {
+  th,
+  td {
+    border: 0;
+    padding: 8px;
+    vertical-align: top;
+  }
+
+  th {
+    text-align: right;
+  }
+}
+
+.work-type-tag {
+  color: #808080;
+}
+
+.work-type {
+  margin-bottom: 18px;
+  margin-top: -8px;
+}
+
+h4 {
+  line-height: 1.3;
```

Comment:
This should be done by setting the bootstrap variable `$font-size-h4` rather than hard coding a value here.  In this way the downstream consumer has a way to override it.